src/messages.cpp

```diff
@@ -224,14 +224,18 @@ const char* FC msgSec( 	// get sub-msg text for system error code
 		{ SECBADFN, "invalid file name" },
 		{ SECBADRV, "invalid drive letter" },
 		{ SECOTHER, "unknown error, errno not set" },
-		{ 32767,    "" }			// terminator
+		{ 32767,    "unhandle system error" }			// terminator
```

Comment:
The correct phrase would be "_unhandled_ system error" (In general, if the word is used as an adjective (to describe something), be aware that it probably should have an "ed" at the end)